voluncheer/profiles/tests/test_views.py

```diff
@@ -74,3 +78,39 @@ def test_signup_form_exists_at_desired_location(self):
         """Tests accessing home by URL"""
         response = self.client.get("")
         self.assertEqual(response.status_code, 200)
+
+
+class OrganizationProfileTest(TestCase):
+    """Test cases for Organization profile based functions"""
+
+    def setUp(self):
+        super().setUp()
+
+    def test_confirm_attendance(self):
+        """Test the confirm attendance feature.
+        The request here is for mimicing the get request of select a volunteer and submit.
+        The first confirm_attendance call won't work since volunteer is not signed up.
+        The second confirm_attendance call will add the volunteer to attended_volunteers.
+        The third confirm_attendance call will remove the volunteer from attended_volunteers.
+        """
+        rf = RequestFactory()
+        get_request = rf.get(
+            f"/attandance/{self.opp.pk}",
+            {"volunteer-attended": [str(self.vol.pk)]},
+        )
+        get_request.user = self.org
+
+        confirm_attendance(get_request, self.opp.pk)
+        self.assertEqual(self.opp.attended_volunteers.count(), 0)
```

Comment:
I don't understand why we are removing volunteers from an opportunity once they have attended it. Organizations are going to want the ability to view the history of their events and this seems counter productive to that. Or do I totally misunderstand what is going on here?